search-app/resources/templates/structure.html

```diff
@@ -0,0 +1,52 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="utf-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <title>{% block title %}{% endblock %}</title>
+
+    {% block head-pre-css %}
+    {% endblock %}
+    <!-- Bootstrap core CSS -->
+    <link rel="stylesheet" href="/site/bootstrap.min.css">
```

Comment:
Reminds me we should work with the Earthdata UI team to update our pages to Earthdata look and feel - CMR-1816.